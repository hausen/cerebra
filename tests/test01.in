# This is a comment
OBS:X,Z,Y
EXO:U1,U2,U3 # So is this

U1 = .7
U2 = .5
U3 = .5

X = U1 + U3 + 1
Z = U2 + 1
Y = X + Z + U1 + U2

OUTPUT 1 : "first output"
        MARGIN: X, Y, Z
        INTERV:

OUTPUT 2 : "second output"
        MARGIN: X, Y 
        INTERV: X = 1  

OUTPUT 3 : "fourth output"
        MARGIN: X, Y 
        INTERV: X = 0 , Z  = 1

OUTPUT 4 : "third output"
        MARGIN: X, Y 
        INTERV: X = 0 , Z  = 1
