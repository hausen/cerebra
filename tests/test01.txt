OBS:X,Z,Y
EXO:U1,U2,U3

U1 = .7
U2 = .5
U3 = .5

X = U1 + U3 + 1
Z = U2 + 1
Y = X + Z + U1 + U2

OUTPUT 1 : "first output"
        MARGIN: X, Y, Z
        INTERV:

OUTPUT 2 : "second output"
        MARGIN: X, Y 
        INTERV: X = 1  

OUTPUT 3 : "third output"
        MARGIN: X, Y 
        INTERV: X = 0 , Z  = 1
