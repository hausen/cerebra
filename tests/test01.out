OUTPUT 1: "first output"
X	Y	Z	Prob.
0	0	0	0.175
0	0	1	0.175
0	1	0	0.075
0	1	1	0.075
1	0	0	0.075
1	0	1	0.075
1	1	0	0.175
1	1	1	0.175
END OUTPUT

OUTPUT 2: "second output"
X	Y	Prob.
0	0	0.350
0	1	0.150
1	0	0.150
1	1	0.350
END OUTPUT

OUTPUT 3: "fourth output"
X	Y	Prob.
0	0	0.350
0	1	0.150
1	0	0.150
1	1	0.350
END OUTPUT

OUTPUT 4: "third output"
X	Y	Prob.
0	0	0.350
0	1	0.150
1	0	0.150
1	1	0.350
END OUTPUT

